### PR TITLE
feat: added  avatar component (#368)

### PR DESCRIPTION
**Pull request by johglove**

### CODE REVIEW

File: src/components/PageContent/Avatar/Avatar.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    "rvt-avatar"
+  ]
+
+  if (size) {    
+    classNameArr.push(parseRivetMultiListUtility(size, Sizes, 'rvt-avatar-'))
+  }
+
+  if (className) {
+    classNameArr.push(className)
+  }
```
Comment (burnumd) [inline src/components/PageContent/Avatar/Avatar.jsx:33]: I'm not sure this array placeholder is necessary. The `classNames` library can handle null/undefined values pretty well.

File: src/components/PageContent/Avatar/Avatar.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  .map((r) => AvatarSize.map((s) => `${s}-${r}`))
+  .flat(1);
+export const Sizes = [...AvatarSize, ...AvatarResp];
+
+Avatar.displayName = "Avatar";
+Avatar.propTypes = {
+  /** Text content display in avatar (not displayed if src provided)*/
+  initials: PropTypes.string,
+  /** The static and responsive size options for the avatar */
+  size: PropTypes.oneOfType([PropTypes.array, PropTypes.oneOf([...Sizes])]),
```
Comment (burnumd) [inline src/components/PageContent/Avatar/Avatar.jsx:63]:
```suggestion
  size: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.oneOf(Sizes)), PropTypes.oneOf(Sizes)]),
```

File: src/components/PageContent/Avatar/Avatar.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  "xs",
+  "sm",
+  "md",
+  "lg",
+  "xl"
+];
+const AvatarResp = ["sm-up", "md-up", "lg-up", "xl-up", "xxl-up"]
+  .map((r) => AvatarSize.map((s) => `${s}-${r}`))
+  .flat(1);
+export const Sizes = [...AvatarSize, ...AvatarResp];
```
Comment (burnumd) [inline src/components/PageContent/Avatar/Avatar.jsx:56]: Did you export sizes with the intent to use it in the tests/documentation to indicate that end-users can use the pre-generated sizes?

Comment (burnumd) [inline src/components/PageContent/Avatar/Avatar.jsx:56]: Only components should be capitalized
```suggestion
const avatarSize = [
  "xs",
  "sm",
  "md",
  "lg",
  "xl"
];
const avatarResp = ["sm-up", "md-up", "lg-up", "xl-up", "xxl-up"]
  .map((r) => avatarSize.map((s) => `${s}-${r}`))
  .flat(1);
export const sizes = [...avatarSize, ...avatarResp];
```

**Review by burnumd — CHANGES_REQUESTED** [review]

**Comment by burnumd** [comment]
As a reminder, commits must follow the [Angular Convention](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#specification) for the release process to work

**Comment by johglove** [comment]
What are the conventions not being followed?

**Review by burnumd — APPROVED** [review]

**Pull request merged**